clis/mysql/tool.gpt

```diff
@@ -0,0 +1,24 @@
+name: mysql
+type: context
+description: Provides the ability to interact with mysql databases.
+share tools: sys.exec
+share credential: github.com/gptscript-ai/credential as cli.mysql.password with MYSQL_PWD as env and password as field and "Please enter your mysql user's password:" as message
+
+#!/bin/bash
+
+if ! command -v mysql; then
+  echo "The user does not have the mysql cli installed or it is not available on the PATH."
+else
+  echo "The user has the mysql cli available. Use it to work with mysql."
+  echo "Always execute mysql statements non-interactively from the CLI using one of two options:"
+  echo '`-e "${SQL_QUERY}"` for single sql statements'
```

Comment:
these backticks need to be escaped, the shell is trying to execute them, this successfully runs on my mac.
```#!/bin/bash

if ! command -v mysql; then
  echo "The user does not have the mysql cli installed or it is not available on the PATH."
else
  mysql_cmd=$(command -v mysql)
  echo "The user has the mysql cli available. Use it to work with mysql."
  echo "Always execute mysql statements non-interactively from the CLI using one of two options:"
  echo '`-e "${SQL_QUERY}"` for single sql statements'
  echo "\`< \/path/to/script.sql\` for longer scripts"
  echo ""
  echo 'The password is pre-set using the `$MYSQL_PWD` environment variable'

  echo "the following text is the help output of the mysql cli:"
  echo "\`\`\`"
  ${mysql_cmd} --help
  echo "\`\`\`"
fi
```